contracts/cloud.yml

```diff
@@ -102,12 +103,23 @@ tags:
       Retrieve data, analyze queries, and get query suggestions.
   - name: Tasks
     description: |
-      Process and analyze your data with [tasks](https://docs.influxdata.com/influxdb/cloud/reference/glossary/#task) in the InfluxDB task engine.
-      With tasks, you can schedule Flux scripts to query, analyze, modify, and act on data.
-      In InfluxDB Cloud, you can create tasks that run [invokable scripts](#tag/Invokable-Scripts)
-      with parameters.
+      Process and analyze your data with [tasks](https://docs.influxdata.com/influxdb/cloud/reference/glossary/#task)
+      in the InfluxDB task engine.
+      Use the `/api/v2/tasks` endpoints to schedule and manage tasks, retry task runs, and retrieve run logs.
+
+      To configure a task, you provide the script and the schedule that the task runs.
+      For examples, see
+      [Create a task](#tag/Data-IO-endpoints/operation/PostTasks).
```

Comment:
```suggestion
      [Create a task](/influxdb/v2.4/api/#operation/PostTasks).
```

Reply:
should we use
{{% INFLUXDB_DOCS_URL %}}
here?

Reply:
> should we use {{% INFLUXDB_DOCS_URL %}} here?

We could, but it would result in an unnecessary roundtrip request. This is an internal link targeting the api-docs.